test/spec/modeler/Linting.spec.js

```diff
@@ -110,7 +110,7 @@ describe('Linting', function() {
   }));
 
 
-  (singleStart ? it.only : it)('example', inject(function(bpmnjs, canvas, eventBus, linting, modeling, propertiesPanel) {
+  (singleStart ? it.only : it.skip)('example', inject(function(bpmnjs, canvas, eventBus, linting, modeling, propertiesPanel) {
```

Comment:
Why skip?

Reply:
```suggestion
  (singleStart ? it.only : it)('example', inject(function(bpmnjs, canvas, eventBus, linting, modeling, propertiesPanel) {
```

Reply:
This is a test case for manual testing.

Reply:
Oops, I've already merged this without `skip` ^^